docs/topics/logging.txt

```diff
@@ -650,13 +687,24 @@ logging module.
 Django's default logging configuration
 ======================================
 
-By default, Django configures the ``django.request`` logger so that all messages
-with ``ERROR`` or ``CRITICAL`` level are sent to :class:`AdminEmailHandler`, as
-long as the :setting:`DEBUG` setting is set to ``False``.
+By default, Django configures the following logging:
+
+When :setting:`DEBUG` is ``True``:
+
+* The ``django`` catch-all logger sends all messages at the ``INFO`` level or
+  higher to the console. Django doesn't make any such logging calls at this
```

Comment:
I'm not sure what you're trying to say by "Django doesn't make any such logging calls at this time." Django does make quite a few logging calls that would be caught (via propagation) by the ``django`` catch-all logger. A quick grep turns up at least `django.contrib.gis` and various modules within `django.db`. It seems to be true that Django never logs anything directly to the `django` logger, but I don't think that's really relevant or worthy of mention; the point of the `django` catch-all is to catch things logged in sub-modules and propagated upwards.